src/streamlink_cli/output/player.py

```diff
@@ -235,8 +235,11 @@ def _open(self):
                 warnings.warn(
                     "\n".join([
                         "The --player argument has been changed and now only takes player path values:",
-                        "  Player paths with spaces don't have to be wrapped in quotation marks anymore.",
-                        "  Custom player arguments need to be set via --player-args.",
+                        "  Player paths with spaces must not be wrapped in additional quotation marks",
```

Comment:
It might sound pedantic (I'm just thinking about interpretation), but does this message mean paths without spaces can be wrapped in quotation marks?

Would it be better if it just read:

`Player paths must not be wrapped in quotation marks`?